test_team_guardiansofpii.py

```diff
@@ -15,6 +15,25 @@ def test_email_address(self):
 
     def test_person(self):
         """Test person functionality"""
+        # Positive Testcases
+        test_string = "The PERSON is Harper Jasmine Margarita Johnson"
+        actual = anonymize_text(test_string, ['PERSON'])
+        expected = "The PERSON is <PERSON>"
+        # Replace the person with <PERSON> to apply custom logic
```

Comment:
The problem here is you have a very complex name.  The spacy model we are using was not trained to detect a name as complex as this.  I would suggest using "Jasmine Johnson" as a name.  You will find the model detects that correctly and there is no need to manually replace your more complicated name.

Retraining the model to detect a name like this is the correct solution but is beyond the scope of this class. 

This should work and replace the name correctly 
```
        # Positive Testcases
        test_string = "The PERSON is Jasmine Johnson"
        actual = anonymize_text(test_string, ['PERSON'])
        expected = "The PERSON is <PERSON>"
        self.assertEqual(expected, actual)
```